gke/standard/zonal/no-channel-to-release-channel/main.tf

```
@@ -0,0 +1,191 @@
+/**
+* Copyright 2025 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# NOTE: Locations are different in these examples to satisfy the tests.
+# These are intended to represent sequential steps on the same cluster.
+
+# [START gke_standard_release_channel_none]
+resource "google_container_cluster" "rc_none" {
+  name     = "cluster-zonal-example-none-to-rc"
+  location = "us-central1-a"
+
+  release_channel {
+    channel = "UNSPECIFIED"
+  }
+
+  remove_default_node_pool = true
+  initial_node_count       = 1
+
+  # Set `deletion_protection` to `true` will ensure that one cannot
+  # accidentally delete this instance by use of Terraform.
+  deletion_protection = false
+}
+
+resource "google_container_node_pool" "rc_none" {
+  name    = "cluster-zonal-example-none-to-rc-np"
+  cluster = google_container_cluster.rc_none.name
```

I recommend using the `id` as that is the full `projects/{{project}}/locations/{{location}}/clusters/{{cluster}}`.  Otherwise you will also want to set the `location` to `google_container_cluster.rc_none.location`

```suggestion
  cluster = google_container_cluster.rc_none.id
```